yaptide/celery/utils/pymc.py

```diff
@@ -293,6 +294,59 @@ def read_file(event: threading.Event,
     send_task_update(simulation_id, task_id, update_key, up_dict)
 
 
+def read_fluka_file(event: threading.Event,
+                    dirpath: Path,
+                    simulation_id: int,
+                    task_id: str,
+                    update_key: str,
+                    timeout_wait_for_file: int = 20,
+                    timeout_wait_for_line: int = 5 * 60,
```

Comment:
Can you add `_s` suffix to the variable name to denote which time unit we deal with ?